### PR TITLE
Register DXC for HLSL to SPIR-V compilation

### PR DESCRIPTION
**Pull request by antiagainst**

### CODE REVIEW

**Comment by antiagainst** [comment]
cc @chaoticbob @ehsannas

File: README.md [hunk 1]
```diff
@@ -40,6 +40,17 @@ glslang is a GLSL and HLSL to SPIR-V compiler, usable both as a command-line too
 * Contact: David Neto <dneto@google.com>
 * Compiler requirements: C++11 compatible compiler (MSVC 2013+ on Windows)
 
+## DXC
+DirectXShaderCompiler (DXC) is Microsoft's next-gen official HLSL compiler, based on LLVM/Clang.
```
Comment (krOoze) [inline README.md]: "DirectX Shader Compiler" seems to be the full name. Should be that way in the header, and the first line?

File: README.md [hunk 2]
```diff
@@ -40,6 +40,17 @@ glslang is a GLSL and HLSL to SPIR-V compiler, usable both as a command-line too
...
+It can also compile HLSL into SPIR-V, contributed by Google.
```
Comment (krOoze) [inline README.md]: Sentence structure feels weird. Maybe "thanks to contribution from Google".

"also" feels superfluous as it was not previously specified it is primarily\originaly HLSL->DXIL compiler.

File: README.md [hunk 3]
```diff
@@ -40,6 +40,17 @@ glslang is a GLSL and HLSL to SPIR-V compiler, usable both as a command-line too
...
+
+* Repository: https://github.com/Microsoft/DirectXShaderCompiler
+* License: University of Illinois Open Source License
+* Contact: Lei Zhang <antiagainst@google.com>
```
Comment (krOoze) [inline README.md]: Might want to specify that (only) SPIR-V matters can go to you.

The Readme states `opencode@microsoft.com` as contact. Maybe add that?

File: README.md [hunk 4]
```diff
@@ -40,6 +40,17 @@ glslang is a GLSL and HLSL to SPIR-V compiler, usable both as a command-line too
...
+* Platform: Windows
+* Compiler requirement: Visual Studio 2017
```
Comment (krOoze) [inline README.md]: The wiki also seems to state support for Visual Studio 2015 Update 3.

File: README.md [hunk 5]
```diff
@@ -40,6 +40,17 @@ glslang is a GLSL and HLSL to SPIR-V compiler, usable both as a command-line too
...
+* Download: https://khr.io/dxcappveyorbuild (rolling release)
```
Comment (krOoze) [inline README.md]: Would it be worth mentioning that **some** version is part of the Windows SDK?

File: README.md [hunk 6]
```diff
@@ -40,6 +40,17 @@ glslang is a GLSL and HLSL to SPIR-V compiler, usable both as a command-line too
 * Contact: David Neto <dneto@google.com>
 * Compiler requirements: C++11 compatible compiler (MSVC 2013+ on Windows)
 
+## DXC
+DirectXShaderCompiler (DXC) is Microsoft's next-gen official HLSL compiler, based on LLVM/Clang.
+It can also compile HLSL into SPIR-V, contributed by Google.
+
+* Repository: https://github.com/Microsoft/DirectXShaderCompiler
```
Comment (krOoze) [inline README.md]: Might want to also add https://github.com/Microsoft/DirectXShaderCompiler/wiki as home\landing page?

File: README.md (hunk 1 above)
Comment (antiagainst) [reply README.md]: Yep. Done.

File: README.md (hunk 2 above)
Comment (antiagainst) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (antiagainst) [reply README.md]: Added.

File: README.md (hunk 4 above)
Comment (antiagainst) [reply README.md]: IIRC, we used to have problems of compiling it using VS 2015. VS 2015 path is not tested, at least publicly.

File: README.md (hunk 5 above)
Comment (antiagainst) [reply README.md]: The version shipped in Windows SDK does not contain SPIR-V CodeGen. I think this doc is regarding Vulkan ecosystem; so I'm not sure it matters to mention that version here.

File: README.md (hunk 6 above)
Comment (antiagainst) [reply README.md]: Done

**Comment by ehsannas** [comment]
👍

**Pull request merged**